docs/architecture/microservices/secure-net-microservices-web-applications/authorization-net-microservices-web-applications.md

```diff
@@ -6,7 +6,7 @@ ms.date: 01/30/2020
 ---
 # About authorization in .NET microservices and web applications
 
-After authentication, ASP.NET Core Web APIs need to authorize access. This process allows a service to make APIs available to some authenticated users, but not to all. [Authorization](/aspnet/core/security/authorization/introduction) can be done based on users’ roles or based on custom policy, which might include inspecting claims or other heuristics.
+After authentication, ASP.NET Core Web APIs need to authorize access. This process allows a service to make APIs available to some authenticated users, but not to all. Authorization can be done based on users’ roles or based on custom policy, which might include inspecting claims or other heuristics.
```

Comment:
I'm confused. The target page is still published here: https://docs.microsoft.com/aspnet/core/security/authorization/introduction

Why did you think this was broken?